includes/blocks/site-automation-block/markup.php

```diff
@@ -8,8 +8,11 @@
  * @var array $curated_posts Curated content from Sophi.
  */
 
+$page_name   = preg_replace( '/\s+/', '-', strtolower( $attributes['pageName'] ) ); // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$widget_name = preg_replace( '/\s+/', '-', strtolower( $attributes['widgetName'] ) ); // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
```

Comment:
This is fine but you could also probably use `sanitize_title`, as I think that does what we want.